docs/sources/operator/architecture.md

```diff
@@ -132,3 +132,69 @@ Two labels are added by default to every metric:
 
 The shard number is not added as a label, as sharding is designed to be
 transparent on the receiver end.
+
+## Enabling sharding and replication
+
+To enable sharding and replication, you need to set the `shards` and `replicas` properties in the Grafana Agent configuration file. For example, the following configuration file would shard the data into 3 shards and replicate each shard to 2 other Grafana Agent instances:
```

Comment:
```suggestion
To enable sharding and replication, you must set the `shards` and `replicas` properties in the Grafana Agent configuration file. For example, the following configuration file would shard the data into three shards and replicate each shard to two other Grafana Agent instances:
```

Single digit numbers should be spelled out: https://developers.google.com/style/numbers